### ISSUE TITLE
BIPM reference format (French only): "e" must be set as superscript in meeting references

### ISSUE THREAD

**Issue by manuelfuenmayor** [relaton/relaton-bipm#63]
From BIPM:
*The form of the references for the meetings needs a minor adjustment in the French bibliography (e.g. p111)*
 
![image001](https://github.com/user-attachments/assets/45e81f88-785b-41f8-a7eb-7d93f3a7c63a)

*Here the “e” should be superscript and the “r” of réunion lower-case (both times): CIPM 43e réunion (1950)… 43e réunion du CIPM and ‘Comité international des poids et mesures’ (capital letter for Comité only)*

The reference above comes from this bibliographic markup:

```
* [[[PV22_h,CIPM Meeting 43]]]
```

Basically:
- "e" must be set as superscript, and
- "Comité International des Poids et Mesures" should be "Comité international des poids et mesures"

In relation to https://github.com/metanorma/bipm-si-brochure/issues/240

**Comment by ronaldtse** [comment]
This requires the organization name to be multilingual so in French we use the French name.

**Comment by manuelfuenmayor** [comment]
@andrew2net I'm building the SI Brochure with version 1.19.3 of relaton-bipm and yet I don't see this issue fixed. Should I do something else?

**Comment by andrew2net** [comment]
@manuelfuenmayor, the relaton-data-bipm repository hasn't been updated yet. There is an [issue](https://github.com/metanorma/bipm-data-outcomes/issues/33) what we need to resolve to update the dataset.

**Comment by andrew2net** [comment]
@manuelfuenmayor please try relaton-bipm v1.19.4 now

**Comment by manuelfuenmayor** [comment]
@andrew2net I'm tried with relation-bipm v.1.20.2 and still the issue persists:
![relaton-output](https://github.com/user-attachments/assets/f1f78730-4c55-485f-afef-b7509eccb5c6)

**Comment by manuelfuenmayor** [comment]
Also, I'm seeing this in the bibliography:
![issue-refs](https://github.com/user-attachments/assets/bbff6805-ea49-4988-9288-8beaa87b8556)

**Comment by andrew2net** [comment]
@ronaldtse can we close this issue?

**Comment by ronaldtse** [comment]
Yes thank you @andrew2net !
